### PR TITLE
Fix Accordion keyboard navigation

### PR DESCRIPTION
**Pull request by mperrotti**
#### Related issues
Fixes https://meetup.atlassian.net/browse/SDS-588

#### Description
Focus styles are visible again, and panels can be toggled using the keyboard. Also did a little cleanup, especially in tests.

#### Screenshots (if applicable)
screen shot 2018-04-24 at 2 30 38 pm

### CODE REVIEW

File: src/interactive/AccordionPanel.jsx [hunk 1]
```diff
@@ -69,9 +71,26 @@ class AccordionPanel extends React.Component {
 	onToggleClick(e) {
 		e.preventDefault();
 
+		// console.log(this.props.onToggleClick ? 'this.props.onToggleClick(e)' : 'this._handleToggle(e);');
```
Comment (akdetrick) [inline src/interactive/AccordionPanel.jsx]: 🔪

File: src/interactive/AccordionPanel.jsx [hunk 2]
```diff
@@ -69,9 +71,26 @@ class AccordionPanel extends React.Component {
...
+
 		this.props.onToggleClick ? this.props.onToggleClick(e) : this._handleToggle(e);
 	}
 
+	/**
+	 * @description forceUpdate allows us to calculate height again now that contentEl is set
```
Comment (akdetrick) [inline src/interactive/AccordionPanel.jsx]: needs a doc update

File: src/interactive/AccordionPanel.jsx [hunk 3]
```diff
@@ -168,9 +187,11 @@ class AccordionPanel extends React.Component {
 					aria-expanded={isOpen}
 					aria-selected={isOpen}
 					className={classNames.trigger}
+					tabIndex={0}
+					onKeyUp={this.onKeyUp}
 					onClick={this.onToggleClick}
-					onFocus={this.handleFocus}
-					onBlur={this.handleBlur}
+					// onFocus={this.handleFocus}
```
Comment (akdetrick) [inline src/interactive/AccordionPanel.jsx]: 🔪

**Comment by mperrotti** [comment]
I fixed all those dumb mistakes before PR, but forgot to push up again 😳

**Pull request merged**